### DISCUSSION TITLE
Trouble configuring openwrt device

### DISCUSSION THREAD

**Discussion by openSourceBugs** [Ysurac/openmptcprouter#3607] [General]
Is RealTek RTL8188CU  802.11n wireless adapter supported in openwrt? It's this item: https://www.amazon.com/dp/B01ERHE18S?ref=ppx_yo2ov_dt_b_fed_asin_title

 I can't seem to get it to function.  I want to use a raspbery pi (cm4) internal wireless, connect a usb wifi adapter to a usb hub and connect the hub to the pi.  But it appears that only the raspberry pi cm4's internal wireless is working.  I can't see any interface/device/etc associated with the realtek usb device.  I am trying to connect it to a hotspot but if the hardware/drivers/linux are just taking a dump due to being unsupported, i dont' want to waste anymore time.

**Comment by openSourceBugs** [comment]
Looking at the realtek website, they only provide drivers for windows.  what a scam of an amazon seller saying linux/pi are supported.  anyways.  Does openwrt allow me to set up only one connection using the internal wireless of the pi or must it use at least two external modems for anything to work?  I tried just setting up the internal wireless to connect to my netgear home router but that did not turn the openwrt cm4 device into a router to provide internet to my laptop.

**Comment by muko82** [comment]
Check this: https://openwrt.org/packages/pkgdata/kmod-rtl8xxxu
Or this: https://openwrt.org/docs/techref/driver.wlan/rtl819x
